src/host/ut_host/VtIoTests.cpp

```diff
@@ -554,12 +557,224 @@ class ::Microsoft::Console::VirtualTerminal::VtIoTests
         actual = readOutput();
         VERIFY_ARE_EQUAL(expected, actual);
 
+        // Copying from a partially out-of-bounds source to a partially out-of-bounds target,

```

Comment:
i friggin love me some diagrams

Reply:
I tried to write a diagram but failed to make it readable & obvious since it requires 3 overlapping rectangles. Instead, I tried to improve the verbal description.

I'll merge for now since we're short on time.

Reply:
Oh, I thought this was getset.cpp haha. Thank you in this case!